### PR TITLE
fetch.spec.js quick fix to pass on Node.js 4 (1.3.x only)

### PR DESCRIPTION
**Pull request by brodybits**
this workaround is needed in 1.3.x since latest browser release dropped support for Node.js 4

### CODE REVIEW

File: spec/fetch.spec.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe('fetch/uninstall tests via npm & git', function () {
             .then(_ => uninstall('cordova-android', tmpDir, opts))
             .then(_ => expectNotToBeInstalled('cordova-android'))
 
-            .then(_ => fetchAndMatch('https://github.com/apache/cordova-browser.git', { name: 'cordova-browser' }))
+            .then(_ => fetchAndMatch('https://github.com/apache/cordova-browser.git#5.0.x', { name: 'cordova-browser' }))
```
Comment (janpio) [inline spec/fetch.spec.js]: Why is this necessary from a technical standpoint?

Comment (raphinesse) [reply spec/fetch.spec.js]: Probably because latest browser release dropped support for Node 4.

**Comment by brodybits** [comment]
I just updated the description, comment in test script, and commit message to include the clarification by @raphinesse.

**Comment by raphinesse** [comment]
Closing this as 1.3.1 has been released. Please undo if this is still needed.
